src/gpu_sysman.c

```diff
@@ -1872,14 +1878,25 @@ static int gpu_config_parse(const char *key, const char *value) {
   } else if (strcasecmp(key, KEY_LOG_GPU_INFO) == 0) {
     config.gpuinfo = IS_TRUE(value);
   } else if (strcasecmp(key, KEY_METRICS_OUTPUT) == 0) {
-    config.output = OUTPUT_UNSET;
-    for (unsigned i = 0; i < STATIC_ARRAY_SIZE(metrics_output); i++) {
-      if (strcasecmp(value, metrics_output[i]) == 0) {
-        config.output = i;
-        break;
+    config.output = 0;
+    static const char delim[] = ",:/ ";
+    char *save, *flag, *flags = strdup(value);
+    for (flag = strtok_r(flags, delim, &save); flag;
+         flag = strtok_r(NULL, delim, &save)) {
+      unsigned i;
+      for (i = 0; i < STATIC_ARRAY_SIZE(metrics_output); i++) {
+        if (strcasecmp(flag, metrics_output[i].name) == 0) {
+          config.output |= metrics_output[i].value;
+          break;
+        }
+      }
+      if (i >= STATIC_ARRAY_SIZE(metrics_output)) {
+        free(flags);
+        return RET_INVALID_CONFIG;
```

Comment:
Should there be a log entry here as well?

Reply:
Added.